sass/directives/_tables.scss

```diff
@@ -178,3 +143,90 @@
     content: '\f00c'; // Checkmark icon
   }
 }
+
+@mixin section-bar($breakpoint) {
+  background-color: $table-row-light;
+  padding: $base-margin / 2;
+  margin-bottom: $section-padding;
+  border-radius: $minor-border-radius;
+  text-indent: $type-indent;
+
+  @media only screen and (max-width: $breakpoint) {
+    margin-bottom: 0;
+  }
+}
+
+@mixin table__row-alternating--colors {
+  tr:nth-child(odd) {
+    th, td {
+      background-color: $table-row-light;
+    }
+
+    .darken {
+      background-color: $table-row-light-promo;
+      font-weight: $font-weight-bold;
+    }
+  }
+
+  tr:nth-child(even) {
+    th, td {
+      background-color: $table-row-dark;
+    }
+
+    .darken {
+      background-color: $table-row-dark-promo;
+      font-weight: $font-weight-bold;
+    }
+  }
+}
+
+@mixin table__column-highlight($highlight-content, $table-highlight-z-index) {
+  &:hover:after,
+  &:active:after,
+  &:focus:after {
+    content: $highlight-content;
+    position: absolute;
+    background-color: rgba($grey-darkest, 0.1);
+    left: 0;
+    top: -5000px;
+    height: 10000px;
+    width: 100%;
+    z-index: $table-highlight-z-index;
+    pointer-events: none;
+  }
+}
+
+@mixin table__content-format {
```

Comment:
this should be called `table--row-column-headers` b/c this is the part that sets up a table with row headers and column headers. Rather than being generic formatting for table content, it's a specific format

then `ecom-table--row-column-headers` could be just `ecom-table`

and you'd rename `table--row-column-headers` to `product-table`

or `table--ecom` and `table--product` would be more BEM-y